package.json

```diff
@@ -87,18 +94,18 @@
     "test:serialize:11": "cd ./tests/serialize && node ./data.js -in=structures.n3 -format=application/rdf+xml  -out=,structures.xml && node diff ,structures.xml t11-ref.xml",
     "test:serialize:12": "cd ./tests/serialize && node ./data.js -in=structures.n3 -format=text/turtle -out=,structures.ttl && node diff ,structures.ttl t12-ref.ttl",
     "test:serialize:13": "cd ./tests/serialize && node ./data.js -in=structures.n3 -format=application/n-triples -out=,structures.nt && node ./data.js -format=application/n-triples -in=,structures.nt -format=text/turtle -out=,structures.nt.ttl && node diff ,structures.nt.ttl t13-ref.ttl",
-    "test:unit": "mocha --growl --require @babel/register tests/unit/**-test.js",
-    "test:unit:egp": "mocha --require @babel/register tests/unit/fetcher-egp-test.js",
-    "test:unit:dev": "mocha --watch --growl --require @babel/register tests/unit/**-test.js"
+    "test:unit": "mocha --growl --require ./tests/babel-register.js tests/unit/**-test.js",
+    "test:unit:egp": "mocha --require ./tests/babel-register.js tests/unit/fetcher-egp-test.js",
+    "test:unit:dev": "mocha --watch --growl --require ./tests/babel-register.js tests/unit/**-test.js"
   },
   "files": [
     ".babelrc",
     "src",
     "lib",
     "dist"
   ],
-  "main": "lib/index.js",
-  "module": "src/index.js",
+  "main": "lib/index",
+  "module": "src/index",
```

Comment:
I don't think the extensions should be removed here? One thing you might want to add, though, is [the `types` property](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package), after [configuring TypeScript to also emit a `--declaration`](https://www.typescriptlang.org/docs/handbook/compiler-options.html).